pages/api/stripe/stripeProducts.ts

```diff
@@ -349,11 +349,12 @@ export function getProductProperties(product) {
   return STRIPE_PRODUCT[product].properties;
 }
 
+//!TODO: Fix the conditional because always is development and the products aren't created in stripe
 export function getStripeProduct(opt) {
   const { product, anonymousId, impactId } = opt;
   const selectedProduct = STRIPE_PRODUCT[product];
-  selectedProduct.session.line_items[0].price =
-    process.env.NODE_ENV === 'production' ? selectedProduct.production : selectedProduct.debug;
+  selectedProduct.session.line_items[0].price = selectedProduct.production;
+  // process.env.NODE_ENV.trim() === 'production' ? selectedProduct.production : selectedProduct.debug;
```

Comment:
why?

Reply:
Because if you run "yarn dev" and in your .env.local you have `NODE_ENV=production`, process.env.NODE_ENV is still in development, so price tests always fail. I have tried adding trim() to make sure it is not the problem.

Reply:
there should not be NODE_ENV set to production on local